proposals/0000-improved-ghc-development.rst

```diff
@@ -0,0 +1,216 @@
+.. proposal-number:: Leave blank. This will be filled in when the proposal is
+                     accepted.
+
+.. trac-ticket:: Leave blank. This will eventually be filled with the Trac
+                 ticket number which will track the progress of the
+                 implementation of the feature.
+
+.. implemented:: Leave blank. This will be filled in with the first GHC version which
+                 implements the described feature.
+
+Improved GHC development
+========================
+
+Developing GHC can be burdonsome, and the whole process has received lots of
+critique. We propose some (potential fundamental) changes to address these
+shortcomings. We believe the current tools supporting GHC development are more
+than adequate but also believe there is room for improvements. These proposals
+all enhance the existing tools instead of replacing them outright.
+
+Motivation
+----------
+
+Make GHC development fun again! What I’d really like to see improved is the “multiple (web)services, multiple accounts” issue solved. As it is right now I need to have a github account (ghc-proposals, soon tiny PR against ghc), a trac account (for the wiki and issue tracker; which never seems to remember me), a phabricator account, likely a stack overflow account for questions and answers. On top of that I also need to be on a large set of mailing lists if I want to not miss anything, and an irc, reddit, and twitter client to tap into the most recent discussions. And there’s another wiki on haskell.org for which I would need to sign up and get an approved account.
+
+The often proposed solution is to move everything to GitHub. Some of us believe GitHub is not up to the task of managing a project such as GHC. But we also believe with minor changes we can greater leverage the user base of GitHub while also providing valuable information back to the non GitHub users.
+
+Proposed Change
+---------------
+- Link Phabricator and GitHub
+
+  Phabricator seems to have built in support for OAuth. One easy way so lower the barrier is to support OAuth logins to GitHub. This way a user doesn't have to create a new account. They can simply login using their GitHub credentials. *(This seems to already be enabled, maybe we should advertise it more.)*
```

Comment:
Indeed, this works today. I can say this with certainty as this is my primary means of logging in.